assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -122,12 +123,7 @@ const buttonDimensions = { width: '24', height: '24' };
 
 const StyledGridViewButton = styled(GridViewButton).attrs(buttonDimensions)``;
 
-const SafeZoneButton = styled(SafeZone).attrs(buttonDimensions)`
-  ${({ active, theme }) =>
-    active &&
-    css`
-      background: ${rgba(theme.colors.bg.white, 0.1)};
-    `}
+const SafeZoneToggle = styled(ToggleButton).attrs(buttonDimensions)`
```

Comment:
Also, the `buttonDimensions` here don't appear to make sense anymore, given that `ToggleButton` expects `iconWidth` and `iconHeight` props instead of `width` and `height`. The common object probably isn't necessary anymore and the two definitions should just be inlined:

```js
const StyledGridViewButton = styled(GridViewButton).attrs({ width: 24, height: 24 })``;
const SafeZoneToggle = styled(ToggleButton).attrs({ iconWidth: 24, iconHeight: 24 })` ...
```

Or alternatively, make the `ToggleButton` use the same props, but that requires refactoring a bunch more stuff I guess.

If changed either way, they can be removed [below](https://github.com/google/web-stories-wp/pull/4262/files#diff-9ca2523a045442855dbdd7218d0095acR479-R480).